tensorrt_llm/bench/benchmark/throughput.py

```diff
@@ -234,10 +234,11 @@
     help="Path where per request information is written to.",
 )
 @optgroup.option(
-    "--enable_chunked_context/--disable_chunked_context",
-    default=True,
-    help=
-    "Enable/disable chunking in prefill stage for enhanced throughput benchmark. "
+    "--enable_chunked_context",
+    is_flag=True,
+    default=None,
```

Comment:
Should this actually be False?

Reply:
so you mean `default=False`? but default is not false for trt right?